api/modules/publishedProjects/controller.js

```diff
@@ -15,6 +15,44 @@ function ensureRemixSuffix(title) {
   return title.replace(/( \(remix\))*$/, ' (remix)');
 }
 
+// Taken from http://stackoverflow.com/a/643827
+function isDate(val) {
```

Comment:
code duplication from  api/modules/projects/controller.js, so this should live in a util file that is required by both controllers.